core/vmpu/inc/vmpu.h

```diff
@@ -161,8 +161,8 @@ extern int vmpu_fault_recovery_bus(uint32_t pc, uint32_t sp, uint32_t fault_addr
 
 uint32_t vmpu_fault_find_acl(uint32_t fault_addr, uint32_t size);
 
-extern void vmpu_acl_stack(uint8_t box_id, uint32_t bss_size, uint32_t stack_size, uint32_t * bss_start,
-                           uint32_t * stack_pointer);
+extern void vmpu_acl_sram(uint8_t box_id, uint32_t bss_size, uint32_t stack_size, uint32_t * bss_start,
```

Comment:
Prefer function names that are verbs.

Was this stacking an ACL region before? Or was it adding the stack to the ACL?

Consider using `vmpu_acl_sram_push`, if there is some stacking semantics. Otherwise, `vmpu_acl_sram_add` would be good.